### DISCUSSION TITLE
Missing .dll after compiling

### DISCUSSION THREAD

**Discussion by HudsonSchumaker** [msys2/MINGW-packages#11523] [Q&A]
Hello, 
after compile(using mingw64 g++) my code when I try to run I get a error message saying: libgcc_s_seh-1.dll is missing.
I set up a git hub action to be more easy to share what I am using and the way I am building.

[action](https://github.com/SchumakerTeamStudios/bullet-hell-jam-2022/blob/main/.github/workflows/build_windows.yml)
Thanks in advance

**Comment by Biswa96** [comment]
If you want to use the binary in msys2 environment you have to execute it in mingw64 environment.
If you want to use the binary as a standalone program you have to copy all the required dependencies i.e. those dlls in that same directory.

**Reply by HudsonSchumaker** [reply]
I see, there is way to put in a lib folder and link ? is too much files in the root

**Reply by Biswa96** [reply]
If you are just testing the program you can run it in mingw64 terminal (blue icon).

**Reply by HudsonSchumaker** [reply]
I mean for distribution
